### PR TITLE
Server response transformation

### PR DESCRIPTION
**Pull request by Kasmadei**

### CODE REVIEW

File: src/models/eventModel.tsx [hunk 1]
```diff
@@ -19,6 +19,10 @@ const ctx = {
   hasFailureRate: VocabularyUtils.PREFIX + "has-failure-rate",
   requirement: VocabularyUtils.PREFIX + "has-requirement",
   upperBound: VocabularyUtils.PREFIX + "to",
+  criticality: VocabularyUtils.PREFIX + "criticality",
+  hasPrediction: VocabularyUtils.PREFIX + "has-prediction",
+  value: VocabularyUtils.PREFIX + "value",
+  behavior: VocabularyUtils.PREFIX + "behavior",
```
Comment (kostobog) [inline src/models/eventModel.tsx:25]: This should be:
 behavior: VocabularyUtils.PREFIX + "**is-manifestation-of**"

Comment (blcham) [reply src/models/eventModel.tsx:25]: @kostobog but i guess it makes sens to have something like:
isManifestationOf: VocabularyUtils.PREFIX + "is-manifestation-of",


I mean, I do not see reason to have different json name and RDF property.

Comment (kostobog) [reply src/models/eventModel.tsx:25]: @blcham I think in json or jopa object `behavior` is more readable than `isManifestationOf`. May be we can change it in the future.

Comment (Kasmadei) [reply src/models/eventModel.tsx:25]: so, i'll keep "behavior", right ?

**Pull request closed without merging**